README.md

```diff
@@ -1,16 +1,28 @@
-# Title
+# Environment Overview
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
++ Describe the purpose of the Learn IDE
++ Explain what a development environment is and what it is used for
```

Comment:
This is super minor, but I would change the order of these so that the second one comes first. It seems like that's how the outline content flows and it makes sense that way.

Reply:
That's helpful - thanks! Will change now